metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -112,20 +114,17 @@ def _print_issue_list_and_exit(
     sys.exit(1)
 
 
-def docker_check_impl() -> None:
-    issues = check_local_docker_containers()
-    if not issues:
-        click.secho("✔ No issues detected", fg="green")
-    else:
-        _print_issue_list_and_exit(issues, "The following issues were detected:")
-
-
 @docker.command()
 @upgrade.check_upgrade
-@telemetry.with_telemetry
+@telemetry.with_telemetry()
 def check() -> None:
     """Check that the Docker containers are healthy"""
```

Comment:
On quickstart, we must call check repeatedly in a loop right?

Reply:
this is the `datahub docker check` command. it and quickstart share an underlying implementation